setup.py

```diff
@@ -730,18 +730,17 @@ def run(self):
                 "-DEXECUTORCH_BUILD_KERNELS_QUANTIZED_AOT=ON",
             ]
 
+            if ShouldBuild.xnnpack():
+                cmake_args += ["-DEXECUTORCH_BUILD_XNNPACK=ON"]
+                build_args += ["--target", "portable_lib"]
```

Comment:
if user explicitly turns of pybind (via `pybind --off`), don't we still want portable_lib?

Reply:
We currently only add portable_lib if pybindings is turned on: https://github.com/pytorch/executorch/blame/97bca0547a224e2df15f823b1d7966e8b7546b70/setup.py#L726-L740

I guess I assumed this was just for XNNPACK, but is portable_lib for *any* pybindings?

Reply:
Based on the build steps, it seems like it's needed for all pybindings: https://github.com/pytorch/executorch/blob/97bca0547a224e2df15f823b1d7966e8b7546b70/CMakeLists.txt#L858

I'm assuming CI isn't catching this because we never use a single non-xnnpack binding.